hack/examples/pipeline-gradle.yaml

```diff
@@ -53,7 +53,7 @@ spec:
         - fetch-repository
       params:
         - name: MAVEN_MIRROR_URL
-          value: http://jvm-build-workspace-artifact-cache.$(context.taskRun.namespace).svc.cluster.local/v1/cache/default/0/
+          value: http://jvm-build-workspace-artifact-cache.$(context.taskRun.namespace).svc.cluster.local
```

Comment:
so the issue isn't only whether or not the `$(context.taskRun.namespace)` is correct or not based on what I was told yesterday

the service you are trying to reach may be in the same logical workspace / namespace, but may be on a different compute cluster

hence the classic `svc.cluster.local` DNS won't work

hence we need ingress / haproxy similar to what Frederic demo'ed for PaC and github webhooks.

Now, short term, with only 1 compute cluster, sure this might help, so for our initial onboarding efforts I think we may be OK.  But we can't wait long to the next step on ingress / haproxy.

Reply:
Then we also need to re-think the bloom filter approach too, as that assumes that the configmap ends up in the same namespace as the cache. 

If we can't really co-locate anything then this is going to be very difficult. 

Also if the cache is now going to be accessible from an external URL we need to re-think its security. We are going to need to add some kind of authentication.

Reply:
It will also be an issue for security, as we need the secrets to be in the same namespace as the cache.

Reply:
yeah when I found this out there was not a lot of detail around how the types where sharded 

I'll attempt to drill down on that next week.

And like I said, regardless, I think this change is positive regardless